src/main/resources/assets/computercraft/lang/de_de.lang

```diff
@@ -43,32 +43,144 @@ chat.computercraft.wired_modem.peripheral_connected=Peripheriegerät "%s" mit de
 chat.computercraft.wired_modem.peripheral_disconnected=Peripheriegerät "%s" vom Netzwerk getrennt
 
 # Command descriptions, usage and any additional messages.
+commands.computercraft.synopsis=Verschiedene Befehle um Computer zu kontrollieren.
+commands.computercraft.desc=Der /computercraft Befehl enthält verschiedene Werkzeuge um Computer zu debuggen, kontrollieren oder mit ihnen zu interagieren.
+
+commands.computercraft.help.synopsis=Zeigt die Hilfe für den angegebenen Befehl
+commands.computercraft.help.desc=
+commands.computercraft.help.usage=[command]
+commands.computercraft.help.no_children=%s hat keine Unterbefehle
+commands.computercraft.help.no_command=Unbekannter Befehl '%s'
+
+commands.computercraft.dump.synopsis=Zeigt den Status eines Computers.
+commands.computercraft.dump.desc=Zeigt den Status aller Computer oder genauere Informationen über einen angegebenen Computer. Du kannst den Computer entweder über seine Instanz ID (z.B. 123), seine Computer ID (z.B. #123) der seinen Namen (z.B. "@Mein Computer") angeben.
+commands.computercraft.dump.usage=[ID]
+commands.computercraft.dump.action=Zeigt mehr Informationen über einen Computer
+
+commands.computercraft.shutdown.synopsis=Fährt den Computer aus der Ferne herunter.
```

Comment:
Fährt ferngesteuert herunter maybe? This sounds like "this can be used from 5km away".

Reply:
Der Computer lässt sich von einem beliebigen Punkt der Welt aus per Command hoch und herunterfahren. Auch außerhalb der geladenen Chunks. Von daher ist das meiner Meinung nach hier schon korrekt.

Reply:
Aber auch direkt vor dem Computer. Da bin ich noch etwas skeptisch.